buy-resell/products/3rd-party/datev-meinfiskal.md

```diff
@@ -1,76 +1,97 @@
 ---
 slug: /posdealers/buy-resell/products/3rd-party/datev-meinfiskal
 title: DATEV MeinFiskal
----
+---fiskaltrust.Portal
 # DATEV MeinFiskal
 
 :::info summary
 
-After reading this, you will know how the integration of DATEV MeinFiskal works and how to set it up.
+After reading this, you will understand the benefits for PosOperators of connecting their fiskaltrust.Account to _DATEV MeinFiskal_ and how PosDealers can set up the integration.
 
 :::
 
 :::caution Austria / France
 
-As DATEV MeinFiskal is only available in Germany, this tutorial does not apply to Austria or France.
+As _DATEV MeinFiskal_ is only available in Germany, this tutorial does not apply to Austria or France.
+
+Please note that the included links in this section lead to DATEV, which only keeps its documentation in German.
 
 :::
 
- "DATEV MeinFiskal" is an integral part of the "fiskaltrust.Sorglos" product bundle.The data is transferred from the fiskaltrust.portal via an automated interface to the "[DATEV MeinFiskal](https://www.meinfiskal.de/)" platform, so the included services "DATEV Kassenarchiv online" and the "DATEV Kassenbuch-Schnittstelle" can be used.
+  [DATEV](https://www.datev.de/) offers fiscalization solutions for companies in Germany. The solutions for electronic cash management of DATEV meet all legal requirements.   
+_[DATEV MeinFiskal](https://www.meinfiskal.de/)_ is an open cloud platform hosted by DATEV. fiskaltrust, PosCreators and providers of TSE (technical security equipment) joined this platform.  
+_DATEV MeinFiskal_ is an integral part of the _fiskaltrust.Carefree_ product bundle. The data is transferred from the fiskaltrust.Portal via an automated interface to the _DATEV MeinFiskal_ platform.  
+
+The _fiskaltrust.Carefree_ product bundle also includes the _[DATEV Kassenarchiv online](https://apps.datev.de/help-center/documents/0904340)_. This service enables additionally to [revision-safe archiving in fiskaltrust's cloud](https://docs.fiskaltrust.cloud/de/docs/posdealers/buy-resell/products/revision-safe-archiving) daily archiving of end-of-day totals, individual records and other tax-relevant documents as an "extended memory" of the PosSystem. 
+By usage of the _DATEV Kassenarchiv online_, your PosOperator reaches a legally compliant kind of higher security:
+* Additional storage to prevent loss of the data 
+* Proof that nobody can change the PosSystem data
+* Accordance with the GoBD
+* Audit-proof archive for the duration of the statutory retention period
+
+Tax consultants and authorities are working on introducing digital workflows to clients. The interface named _[DATEV Kassenbuch-Schnittstelle](https://apps.datev.de/help-center/documents/0904388)_is available from DATEV; data from PosSystems for financial accounting can be forwarded directly to the tax advisor's DATEV software solution. In addition, an up-to-date database without delays creates transparency in the event of an upcoming external audit.

```

Comment:
_[DATEV Kassenbuch-Schnittstelle](https://apps.datev.de/help-center/documents/0904388)_is 
This is not italicized as you are missing space. In long term it is preferred to use astrix(*) instead of underscore(_) for making italic text, as underscore can't recognize middle of the word.

![image](https://user-images.githubusercontent.com/67687528/187871206-09c80d9d-c740-43d7-ac4c-974cdda313f5.png)